app/models/media_object.rb

```diff
@@ -344,6 +344,7 @@ def to_solr(include_child_fields: false)
       #Add all searchable fields to the all_text_timv field
       all_text_values = []
       all_text_values << solr_doc["title_tesi"]
+      all_text_values << solr_doc["alternative_title_ssim"]
```

Comment:
This looks good.

Leaving a note here for later that this will require a reindexing in order to make the field available for searching.  This could be a limited reindexing...something like:
```
MediaObject.where('alternative_title_ssim:[* TO *]').each(&:update_index)
```